packages/vscode/src/notifications.ts

```diff
@@ -0,0 +1,20 @@
+import { workspace, window, env, Uri, ConfigurationTarget } from 'vscode'
+
+export const showPDPNotification = async () => {
+  const pdpNotif = workspace.getConfiguration('prisma').get('pdpNotification', true)
+
+  if (pdpNotif) {
+    void window
+      .showInformationMessage(
+        'Supercharge your Prisma ORM usage with global database caching, serverless connection pooling and real-time database events.',
+        'Visit Prisma Data Platform',
+      )
+      .then((v) => {
+        if (v === undefined) return
+
+        void env.openExternal(Uri.parse('https://pris.ly/cli/pdp'))
+      })
+  }
+
+  await workspace.getConfiguration('prisma').update('pdpNotification', false, ConfigurationTarget.Global)
```

Comment:
Could be placed in the `if` I think because there is no need to update it again (outside the `if`)

Reply:
Ah yeah, I see what you mean 👍